app/helpers/application_helper.rb

```diff
@@ -3,11 +3,11 @@ module ApplicationHelper
   def site_navigation
     # TODO refactor
     content_tag :ul, class: "nav navbar-nav" do
-      concat navigation_link(t('pages.screencast_list'), screencasts_path, controller: "screencasts")
-      concat navigation_link(t('pages.clip_list'), clips_path, controller: "clips")
+      concat navigation_link(t('pages.screencast_list'), screencasts_path, activeness_criterias: { controller: "screencasts" })
```

Comment:
Line is too long. [127/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.